src/views/v2/month/calendar-body/day.php

```diff
@@ -88,15 +100,15 @@ class="tribe-events-calendar-month__day-date-daynum"
 		<?php if ( ! empty( $day['featured_events'] ) ): ?>
 			<em
 				class="tribe-events-calendar-month__mobile-events-icon tribe-events-calendar-month__mobile-events-icon--featured"
-				aria-label="<?php esc_attr_e( 'Has featured events', 'the-events-calendar' ); ?>"
-				title="<?php esc_attr_e( 'Has featured events', 'the-events-calendar' ); ?>"
+				aria-label="<?php echo esc_attr( sprintf( __( 'Has featured %s', 'the-events-calendar' ), $events_label_plural ) ); ?>"
```

Comment:
:no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: c617965ffdadb9875450f041fd9c405a</sub>